green/test/test_windows.py

```diff
@@ -23,3 +23,6 @@ def test_colorOutput(self):
         self.assertTrue(issubclass(type(gs.stream),
                         colorama.ansitowin32.StreamWrapper))
 
+    def test_unicode_override(self):
+        pass
```

Comment:
The true test is to spawn subprocess of green running that test method, and check the output.

https://github.com/wikimedia/pywikibot-core/blob/master/tests/pwb_tests.py#L71

There all sorts of gotchas, especially when doing this on Windows, and only Python versions, resulting a mess like
https://github.com/wikimedia/pywikibot-core/blob/master/tests/utils.py#L771

However I suspect you'll only need to set `PYTHONIOENCODING` to utf-8 for the subprocess